src/components/Logs.tsx

```diff
@@ -11,11 +12,10 @@ interface Props {
 
 const NEW_LINE = '\r\n';
 
-function Logs(props: Props) {
-    const { token, defaultMessage } = props;
+function Logs({ token, defaultMessage }: Props) {
+    const supabaseClient = useClient();
```

Comment:
Since we're running our calls through a SWR package we should probably use their client hook to be safe.